cves/CVE-2012-3439.yml

```diff
@@ -44,21 +44,21 @@ description_instructions: |
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
 description: | 
-  This entry involved several issues that allowed the authenticator to be bypassed.
-  Prior to the fix, client nonces (random number issued for authentication) were 
-  tracked instead of server nonces, allowing attackers to bypass authentication by 
-  reusing valid requests. Replay attacks (attacks where valid requests are repeated 
-  or delayed) were made possible because the system cached information about the
-  user in the sessision state and did not check for correct credentials and stale
-  nonce values. 
+  Due to several issues authentication could be defeated. First, rather than 
+  server random indentification numbers or "nonces" being tracked. The 
```

Comment:
Typo in identification

Reply:
Also I believe it should be a comma at the end of this sentence rather than a period